Source/ImagePickerController.swift

```diff
@@ -441,7 +441,9 @@ extension ImagePickerController: CameraViewDelegate {
 
   func applyOrientationTransforms() {
     Helper.setOrientation()
-    cameraController.setPreviewLayerOrientation()
+
+    let isiPad = UIDevice.current.userInterfaceIdiom == .pad
+    isiPad ? cameraController.setPreviewLayerOrientation() : ()
```

Comment:
If  UIDevice.current.userInterfaceIdiom == .pad {
   cameraController.setPreviewLayerOrientation()
}

this would be cleaner I think ^